server/services/fileManager.ts

```diff
@@ -0,0 +1,34 @@
+import b2 from 'backblaze-b2';
+import fs from 'fs';
+
+export default class FileManager {
+  protected b2: b2;
+
+  constructor() {
+    this.b2 = new b2({
+      applicationKeyId: process.env.BACKBLAZE_APP_KEY_ID as string,
```

Comment:
Can you update the post controller test to mock this class? It's failing because of 
```FAIL controllers/v1/tests/post.test.ts
  ● Test suite failed to run

    Invalid accountId or applicationKeyId
```

Reply:
lets add the keys as part of the test process via secrets. The test doesn't hit backblaze anyway

Reply:
Ok. Can you update the github actions to set the env variables from secrets? I'll setup the secrets on github after